azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/service/calling/sdk/CallingStateWrapper.kt

```diff
@@ -5,4 +5,8 @@ package com.azure.android.communication.ui.service.calling.sdk
 
 import com.azure.android.communication.calling.CallState
 
-internal data class CallingStateWrapper(val callState: CallState, val callEndReason: Int)
+internal data class CallingStateWrapper(
+    val callState: CallState,
+    val callEndReason: Int,
+    val callEndReasonExtra: Int = 0
```

Comment:
Would recommend "Code" instead of "Extra" here to maintain cohesion with the Calling SDK